recipes/glactools/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.0.9" %} # Remember to update sha256 below
+package:
+  name: glactools
+  version: {{ version }}
+
+build:
+  noarch: generic
+  number: 0
+  script: mkdir -p ${PREFIX}/bin && cp glactools ${PREFIX}/bin && chmod +x $PREFIX/bin/glactools
+
+source:
+  url: https://github.com/grenaud/glactools/releases/download/v1.0.9/glactools
+  sha256: fb9ee6ccb3b02674950e00b163ac69ae077825a574685f4e0f473f6f0a79686f
+
+requirements:
+  build:
```

Comment:
It is recommended to compile your package and not ship pre-build binaries. Can you adopt this please?

Reply:
sure @bgruening